toolbar_utils.py

```diff
@@ -152,7 +151,7 @@ def image_factory(image, toolbar, tooltip=None):
 def spin_factory(default, min, max, callback, toolbar):
     spin_adj = Gtk.Adjustment(default, min, max, 1, 32, 0)
     spin = Gtk.SpinButton(spin_adj, 0, 0)
-    spin_id = spin.connect('value-changed', callback)
+    # spin_id = spin.connect('value-changed', callback)
```

Comment:
This change is inexplicable given the purpose of the pull request.  I note that spin_factory isn't used in this activity, and the source file is used in at least 15 other activities, and spin_factory is used by only one.